### ISSUE TITLE
False positive on clojure.core.aget

### ISSUE THREAD

**Issue by FredrikMeyer** [clj-kondo/clj-kondo#1997]
**version**

clj-kondo v2023.01.20

**platform**

Installed via brew

**problem**

I get an error here:
```
(let [arr (to-array-2d [[1 2] [3 4]])
        test-aget (aget arr 0 1)]
    test-aget)
```

Namely: _clojure.core/aget is called with 3 args but expects 2_

```
> clj-kondo --lint src/cljkondobug/core.clj                                                             ─╯
src/cljkondobug/core.clj:7:19: error: clojure.core/aget is called with 3 args but expects 2
linting took 70ms, errors: 1, warnings: 0
```

**config**

No config.

**expected behavior**

No error.

**thanks**

Thank you for the linter!

**Comment by borkdude** [comment]
Thanks! That's interesting, I'll look into it. You can ignore the bug with `#_:clj-kondo/ignore (aget ...)` for now.

**Comment by FredrikMeyer** [comment]
Thanks :)

It worked in version `2023.01.16`, by the way. (I've pinned clj-kondo to that version in my Github Actions).
